packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -667,12 +674,17 @@ class RenderTableViewport extends RenderTwoDimensionalViewport {
   }) {
```

Comment:
Could you explain to me how the padding is taken into account when calculating the `start` and `end` (input to this method) cells?

Reply:
_Span.trailingOffset is used to calculate the start and end, which includes the padding. There are even tests for this case. Awesome!